### DISCUSSION TITLE
Prevent T00:00:00.000+00:00 being added to dates

### DISCUSSION THREAD

**Discussion by dcoales** [RafaelGB/obsidian-db-folder#861] [Q&A]
Hi,

I have a column defined as a "Calendar" and not a "Calendar Time" and yet whenever I double click and use the date picker to set a date the value in my metadata has T00:00:00.000+00:00 added to the end.  This prevents the value being detected as a date by the Projects plugin (and also looks ugly in dataview queries etc).

Is there any way to stop this happening ?

Thanks,
David

**Comment by reorganice** [comment]
Hi !
Yes, I totally want this feature aswell 💯

Best would be an option in the main settings that could toggle the format of the date-picker globally.

Thank you !!

**Reply by reorganice** [reply]
not happening because of other problems... :/
https://github.com/RafaelGB/obsidian-db-folder/issues/766
